### ISSUE TITLE
why do we need replace_batchnorm ?

### ISSUE THREAD

**Issue by TranThanh96** [THU-MIG/RepViT#32]
why do you need replace_batchnorm  when eval?

**Comment by Isalia20** [comment]
Not the author of the repo but I assume it's done to achieve maximum performance on IoT devices. Batch norm is fused with convolution operation and is done with just convolution(by fusing the conv weights with batchnorm). It first fuses batch norm with convolution and then replaces batch norm layer with identity layer(multiply by 1 basically) to speed up the computation
